starter-code/src/functions-and-arrays.js

```diff
@@ -9,7 +13,28 @@ var words = [
   'pearl',
   'orchard',
   'crackpot'
-];
+]; 
+function findLongestWord(words){
+  for(var i = 0; i < words.length; i++) {
+		var longest = words[i].length;
```

Comment:
You can't assign the var longest inside the for, cause you'll be reasigning it every iteration. I leave you a good example of how it should be done!
```
function findLongestWord(words) {
    var longest = "";
    for(var i = 0; i < words.length; i++) {
        if(words[i].length > longest.length) {
            longest = words[i]
        }
    }
    if(longest === ""){
        return undefined;
    } else {
        return longest;
    }
}
```